packages/synapse-react-client/src/components/SubscriptionPage/SubscriptionPage.tsx

```diff
@@ -31,10 +33,16 @@ export default function SubscriptionPage() {
   }
 
   const { data, hasNextPage, isFetchingNextPage, fetchNextPage } =
-    useGetAllSubscriptions(
+    useGetAllSubscriptions<InfiniteData<Subscription>>(
       currentFilter,
       // Since the user may edit their subscriptions on this page, set staleTime to infinity to prevent re-fetching while editing
-      { staleTime: Infinity },
+      {
+        staleTime: Infinity,
+        select: data => ({
+          pages: data.pages.flatMap(page => page.results),
+          pageParams: data.pageParams,
+        }),
```

Comment:
This `select` was in the custom hook, but I pulled it out to keep the custom hook generic and simple. The generic argument describes the data that the query will return after the `select` transformation.